app/serializers/rest/mute_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class REST::MuteSerializer < ActiveModel::Serializer
+  include RoutingHelper
+
+  attributes :id, :account, :target_account, :created_at, :hide_notifications
+
+  def account
```

Comment:
Kinda of questionable whether this is necessary given that you can only ever look at your own mutes?

Reply:
I removed the account from the serializer.